docs/developer-guide/pull-requests.md

```diff
@@ -0,0 +1,13 @@
+# Reviewing pull requests
+
+-   Use the [GitHub review system](https://help.github.com/articles/about-pull-request-reviews/).
+-   Review against the [Developer Guide criteria](rules.md).
+-   Assign one or more of the appropriate [`PR: needs *` labels](https://github.com/stylelint/stylelint/labels) when requesting a change.
+-   Merge simple documentation fixes after one approval and all other PRs after two approvals.
```

Comment:
We could also add:

-   Keep git history clean:
    -   Start a commit message with uppercase letter.
    -   Consider squash and merge commits instead simple merge. It's useful when PR has “Fix lint error,” “Fix typo,” “Update test snapshot”, etc.
    -   Use [git rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) in your own PRs to avoid unnecessary “Merge ...” commits.

Reply:
Good idea. I've added bits about commit messages, squashing and rebasing.